libcontainer/dmz/Makefile

```diff
@@ -0,0 +1,6 @@
+# Get CC values for cross-compilation.
+include ../../cc_platform.mk
+
+runc-dmz: _dmz.c
```

Comment:
Can we use `diet` here, if available?

Something like
```makefile
DIET ?= $(shell diet echo diet 2>/dev/null || true)

runc-dmz: _dmz.c
        $(DIET) $(CC) $(CFLAGS) -static -o $@ $^
...
```

In my setup, it helps to reduce the size of the binary from 654K to 9.5K without doing anything (except for having `diet` installed). Distro vendors who pack diet may benefit from it (or they can set CC=`diet gcc` but requires an extra step).

Reply:
http://www.fefe.de/dietlibc/

Reply:
I've just checked that diet (package name is `dietlibc-dev`) is available in Debian and Ubuntu, so we can build the official binaries using it. Doesn't have to be implicit if you don't like it that way.

Reply:
How does it work with cross-compilation? Would `diet s390x-linux-gnu-gcc` work? Presumably you'd need `s390x-linux-gnu-diet s390x-linux-gnu-gcc` or something similar.

Reply:
The diet stuff should be a separate PR.
And probably we should rather rewrite the dmz in asm.
Distros don't want to depend on yet another libc.